tests/test-a101-pins.js

```diff
@@ -26,18 +26,35 @@ for(var i = 0; i < GPIOPins.length; i++) {
     // IO6 and IO9 are defined but unusable as GPIOs currently
     if (pinName == "IO6" || pinName == "IO9") continue;
 
-    var pin = gpio.open({ pin: pins[pinName] });
-    var pinValue = pin.read();
-    assert(typeof pinValue == "boolean",
+    var pinA = gpio.open({ pin: pins[pinName], direction: "in" });
+    var pinAValue1 = pinA.read();
+    var ApullValue = (pinAValue1) ? " weak pull-up" : " weak pull-down";
+
+    assert(typeof pinAValue1 == "boolean",
+           "Arduino101Pins: " + pinName + ApullValue + " on input");
+
+    pinA.write(false);
+    var pinAValue2 = pinA.read();
+
+    assert(typeof pinAValue2 == "boolean" && pinAValue1 == pinAValue2,
            "Arduino101Pins: " + pinName + " input");
```

Comment:
@grgustaf If input pin, pin value can not be changed after writing value. Otherwise it must be output pin. So, I write value that is different of default pin value. Usually we set weak pull up on input pins and weak pull down on output pins. Of course, I've seen different settings(like stm32 board). If set weak pull up on input pins, the pin default value must be 'true'. So, I write a fixed value as 'false' to check pin value unchanged.
    In this case, I find that default value of IO3 IO5 IO13 pins are set as 'false' different of other GPIO pins on input pin. Why? PLZ tell me.